services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/DeviceRegistrationInformationService.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.deviceregistry.service.device;
+
+import org.eclipse.hono.util.RegistrationResult;
+
+import io.opentracing.Span;
+import io.vertx.core.Future;
+
+/**
+ * A service which provides device registration information to internal service implementations.
+ */
+public interface DeviceRegistrationInformationService {
```

Comment:
I do not see why we need this additional interface. Does it provide anything different from what the existing `RegistrationService` does?

Reply:
See above (https://github.com/eclipse/hono/pull/2094#discussion_r542476548).